src/main/webapp/features/shared/services/game/Game.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+angular.module('catan')
+        .factory('Game', ['GameModel', '$q', 'Remote', function (GameModel, $q, Remote) {
+
+            var Game = function(idOrDetails) {
+                return new GameModel(idOrDetails);
+            };
+
+            Game.findPublic = function() {
+                return this.findByRemoteService("game", "listPublic");
+            };
+
+            Game.findCurrent = function() {
+                return this.findByRemoteService("game", "listCurrent");
+            };
+
+            Game.findByRemoteService = function(remoteGroup, remoteRequest) {
```

Comment:
but then we will have to use it like this  <ct-game-list  type-of-games="listCurrent">
it looks worse than <ct-game-list type-of-games="CURRENT"> 

but I don't like it as well